src/file_browser.rs

```diff
@@ -385,7 +385,13 @@ impl FileBrowser {
             self.core.get_sender().send(Events::InputEnabled(false))?;
             self.core.screen.suspend().log();
 
-            let status = std::process::Command::new("xdg-open")
+            let cmd = self.core
```

Comment:
You can remove the calls to clone here. If it complains about lifetimes you can do it like this:
```
let config = self.core
                .config.read()?
                .get()?;

Command::new(&config.open_cmd)
```